packages/medusa/src/api-v2/admin/tax-rates/middlewares.ts

```diff
@@ -36,4 +38,14 @@ export const adminTaxRateRoutesMiddlewares: MiddlewareRoute[] = [
       ),
     ],
   },
+  {
+    method: "POST",
+    matcher: "/admin/tax-rates/:id/rules/batch/set",
```

Comment:
question: Do we need the `/set`?

Reply:
Also as a general rule, do we want to have `batch` operations on a separate route from the non-batch ones, or should it be the same, but the data format will differ?
Eg. you can just have it as `:id/rules` and the data can be `data: {}` or `data: [{}]`?

Reply:
> question: Do we need the /set?

The idea here was that we could have different batch operations e.g.: `/set`, `/add`, `/remove` - but maybe there's a better approach.

Reply:
I think the HTTP methods are sufficient to distinguish between those actions, but we can always revisit if that's no longer true